docs/sources/setup/install/helm/install-scalable/aws.md

```diff
@@ -0,0 +1,578 @@
+
+1. Add the policy to the bucket:
+
+    ```bash
+    aws s3api put-bucket-policy --bucket <Your S3 bucket eg. `loki-aws-dev-ruler`> --policy file://bucket-policy-ruler.json
+    ```
+
+## Deploying the Helm chart
+
+1. Add the Grafana chart repository to Helm:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    ```
+1. Update the chart repository:
+
+    ```bash
+    helm repo update
+    ```
+    
+
+### Loki Helm chart configuration
+
+Create a `values.yaml` file with the following content:
+
+```yaml
+ loki:
+   schemaConfig:
+     configs:
+       - from: 2024-04-01
+         store: tsdb
+         object_store: s3
+         schema: v13
+         index:
+           prefix: loki_index_
+           period: 24h
+   storage_config:
+     aws:
+       region: <Insert s3 bucket region> # eu-west-2
+       bucketnames: <Insert s3 bucket name> # Your actual S3 bucket name (loki-aws-dev-chunks)
+       s3forcepathstyle: false
+   ingester:
+       chunk_encoding: snappy
+   pattern_ingester:
+       enabled: true
+   limits_config:
+     allow_structured_metadata: true
+     volume_enabled: true
+     retention_period: 672h # 28 days retention
+   compactor:
+     retention_enabled: true 
+     delete_request_store: s3
+   ruler:
+    enable_api: true
+    storage:
+      type: s3
+      s3:
+        region: eu-west-2
+        bucketnames: <Insert s3 bucket name> # Your actual S3 bucket name (loki-aws-dev-ruler)
+        s3forcepathstyle: false
+      alertmanager_url: http://prom:9093 # The URL of the Alertmanager to send alerts (Prometheus, Mimir, etc.)
+
+   querier:
+     max_concurrent: 4
+
+   storage:
+     type: s3
+     bucketNames:
+       chunks: "<Insert s3 bucket name>" # Your actual S3 bucket name (loki-aws-dev-chunks)
+       ruler: "<Insert s3 bucket name>" # Your actual S3 bucket name (loki-aws-dev-ruler)
+       # admin: "<Insert s3 bucket name>" # Your actual S3 bucket name (loki-aws-dev-admin) - GEL customers only
+     s3:
+       region: <Insert s3 bucket region> # eu-west-2
+       #insecure: false
+     # s3forcepathstyle: false
+       
+       
+ deploymentMode: SimpleScalable
+
+ serviceAccount:
+   create: true
+   annotations:
+     "eks.amazonaws.com/role-arn": "arn:aws:iam::<Account ID>:role/LokiServiceAccountRole" # The service role you created
+
+ backend:
+   replicas: 2
+   persistence:
+     storageClass: gp2
+     accessModes:
+       - ReadWriteOnce
+     size: 10Gi
+ read:
+   replicas: 2
+   persistence:
+     storageClass: gp2
+     accessModes:
+       - ReadWriteOnce
+     size: 10Gi
+ write:
+   replicas: 3
+   persistence:
+     storageClass: gp2
+     accessModes:
+       - ReadWriteOnce
+     size: 10Gi
+
+ # This exposes the Loki gateway so it can be written to and queried externaly
+ gateway:
+   service:
+     type: LoadBalancer
+
+
+ # Enable minio for storage
+ minio:
+   enabled: false
+ 
+```
+
+{{< admonition type="caution" >}}
```

Comment:
haha but good.  I've never blindly copypastad something off of a website before. Why are you looking at me like that?